rotkehlchen/data_import/utils.py

```diff
@@ -27,12 +35,26 @@ def __init__(self, db: DBHandler) -> None:
         self._margin_trades: list[MarginPosition] = []
         self._asset_movements: list[AssetMovement] = []
         self._history_events: list[HistoryBaseEntry] = []
+        self.total_entries: int = 0
+        self.imported_entries: int = 0
+        self.import_msgs: list[dict] = []
+        self.max_msgs = False
+        self.name: str = ''
 
     def import_csv(self, filepath: Path, **kwargs: Any) -> tuple[bool, str]:
         try:
             with self.db.user_write() as write_cursor:
                 self._import_csv(write_cursor, filepath=filepath, **kwargs)
```

Comment:
we should reset the `self.total_entries` and others before calling `self._import_csv` again right? because what if someone imports another CSV again?